### DISCUSSION TITLE
Self-nominating as a Node Maintainer

### DISCUSSION THREAD

**Discussion by TisVictress** [paketo-buildpacks/feedback#129] [Node.js Team]
I'd like to nominate myself to become a Nodejs maintainer. I recently contributed to a few nodejs projects for the [Disable SBOM](https://github.com/paketo-buildpacks/nodejs/issues/568) initiative.

@paketo-buildpacks/nodejs-maintainers 👍 / 👎 ?

**Comment by ryanmoran** [comment]
Congrats @TisVictress on becoming a Node.js Maintainer!! 🎉
